### DISCUSSION TITLE
timeout telnet

### DISCUSSION THREAD

**Discussion by amirmahdi-chiti** [kube-hetzner/terraform-hcloud-kube-hetzner#1050] [Q&A] [answered]
hi guys
on opensuse i can not establish telnet connection to any server. when i enter telnet command , i give connection timed out
but i can ping those servers.
what should i do ? 
(i'm using defualt config of kube.tf)
please help me

**Comment by Silvest89** [comment, marked as answer]
You should check out the firewall rules part. By default only few are allowed, SSH port, http/https (80/443) etc. Add another for telnet port
